Servers/UI/OJS.Servers.Ui/ClientApp/src/components/home-contests/contest-card/ContestCard.tsx

```diff
@@ -34,6 +37,8 @@ const ContestCard = ({ contest }: IContestCardProps) => {
     const contestCardControlBtns = 'card-control-buttons';
     const contestCardControlBtnsClassName = concatClassNames(styles.contestCardControls, contestCardControlBtns);
 
+    const { getRegisterContestTypeUrl } = useUrls();
```

Comment:
This should be in a new hook called `useAppUrls` or something.

`useUrls` is for API calls only.

Reply:
Resolved in a different issue.